docs/src/adr/019adr-changelog-management.md

```diff
@@ -0,0 +1,182 @@
+- Infrastructure maintainers have to learn how to configure and maintain the tool.
+- We pickup a dependency external to both the company and our project
+- Changelogs are meant to communicate a very specific kind of information to a
+  specific audience. Git commit messages, pull requests, and issues are for
+  communicating quite different information to an entirely different audience.
+  Requiring that these communication channels be overloaded risks degrading the
+  quality of communications in all the conflated channels.
+  - However, it's possible we could work out some conventions to add empty
+    commits or designate special PR labels to work around this interference?
+
+
+### 2. Use Unclog
+
+@thanethompson has written the [unclog][] utility, which is used in both
+[informalsystems/tendermint-rs][] and [informalsystems/ibc-rs][]. The tool
+stores changelog entries in a directory structure, with each entry in its own
+file, ensuring merge conflicts can't arise. 
+
+Entries are added via the CLI. E.g.:
+
+```sh
+unclog add --id some-new-feature \
+  --issue 23 \
+  --section breaking-changes \
+  --message "Some *new* feature"
+```
+
+If the `--messages` argument is omitted, it will open your configured editor for
+authoring the message.
+
+The tool has subcommands to generate and update the `CHANGES.md`, and it supports
+a variety of options via its TOML configuration.
+
+[unclog]: https://github.com/informalsystems/unclog
+[informalsystems/tendermint-rs]: https://github.com/informalsystems/tendermint-rs
+[informalsystems/ibc-rs]: https://github.com/informalsystems/ibc-rs
+
+#### Pros
+
+- Requires no new development
+- It is developed internally to informal
+
+#### Cons
+
+- Adds cognitive and procedural overhead to adding changelog entries
+- Would add a rust dependency to our devenv
+- Requires external contributors to either install a CLI tool or figure out a
+  relatively complex file and folder structure to add changelog entries
+
+### 3. Use a custom git merge driver
+
+If we wrote a custom merge driver to work on the simple changelog format we
+could continue our current practice, and just fall back to the custom merge
+driver in case of merge conflicts.
+
+#### Pros
+
+- Allows us to keep our current process of simply updating a markdown file,
+  with which, moreover, most devs are already familiar
+
+#### Cons
+
+- Could require external dependencies to be installed, depending on
+  implementation.
+- It's not possible to configure custom merge drivers for github, so we'd either
+  need to develop a github action or bot to monitor for merge conflicts in the
+  changelog and merge them automatically, or devs would be back to having to
+  resolve merge conflicts locally.
+
+### 4. Write some lightweight tooling
+
+We could author some simple tooling that would capture the core behavior of
+`unclog`, using files in directories, but neither require external dependencies
+nor knowledge of a new CLI.
```

Comment:
> but neither require external dependencies

That depends very much on the implementation?

> nor knowledge of a new CLI

I don't get this part. If we introduce new tooling, we would need to learn a new CLI? Or did you mean relative to `unclog`?

Reply:
Now that I've read the whole document, this should probably refer down to `### Design`.